### PR TITLE
feat: custom status crud

### PR DESCRIPTION
**Pull request by camilamiz**
## PR Description
This PR adds the documentation for Custom Status related requests.
Related PR: https://github.com/callbellchat/callbell/pull/3004

### CODE REVIEW

File: docs/api/reference/custom_status_api/delete_custom_status.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: DELETE /custom_statuses/:id
+sidebar_position: 6
+---
+
+import RequestTabs from "@site/src/components/Requests/RequestTabs"
+
+# DELETE /contacts/:uuid
```
Comment (ahmedBassell) [inline docs/api/reference/custom_status_api/delete_custom_status.md]: Is this a typo?

File: docs/api/reference/custom_status_api/get_custom_status.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+title: GET /custom_statuses/:id
+sidebar_position: 2
+---
+
+import RequestTabs from "@site/src/components/Requests/RequestTabs"
+
+# GET /custom_statuses
+
+Get a specific custom status given an `id`.
```
Comment (ahmedBassell) [inline docs/api/reference/custom_status_api/get_custom_status.md:10]: Should we add the param `/:id` here?

**Review by ahmedBassell — APPROVED** [review]
LGTM 🙌

File: docs/api/reference/custom_status_api/delete_custom_status.md (hunk 1 above)
Comment (camilamiz) [reply docs/api/reference/custom_status_api/delete_custom_status.md]: It is! Good catch!

**Pull request merged**